qiskit/ignis/experiments/calibration/instruction_data/database.py

```diff
@@ -27,32 +32,33 @@ class PulseTable:
     Each entry of this database represents a single parameter associated with the specific pulse,
     and the pulse template is stored in another relational database.
 
-    For example, database entries may look like:
-
-    ```
-    ========================================================================
-     qubits | channel | gate_type | name | value | validation | timestamp
-    --------+---------+-----------+------+-------+------------+-------------
-     (0,)   | d0      | x90p      | amp  | 0.01  | pass       | 2020.01.01
-    --------+---------+-----------+------+-------+------------+-------------
-     (0,)   | d0      | x90p      | amp  | 0.03  | fail       | 2020.01.02
-    --------+---------+-----------+------+-------+------------+-------------
-     (0,1)  | u0      | cr90p     | amp  | 0.04  | pass       | 2020.01.01
-    ------------------------------------------------------------------------
-    ```
-
     You can search for the specific entry by filtering or you can directly generate
     keyword argument for the target pulse factory.
     """
+    TABLE_COLS = ['qubits', 'channel', 'inst_name', 'stretch', 'pulse_type',
```

Comment:
```suggestion
    TABLE_COLS = ['qubits', 'channel', 'inst_name', 'stretch_factor', 'pulse_type',
```
`stretch` by itself is a little unclear.